lib/module.js

```diff
@@ -10,117 +10,128 @@ const { promisify } = require('util')
 const { hostname } = require('os')
 
 module.exports = function module (moduleOptions) {
-  const defaults = {
-    path: 'sitemap.xml',
-    hostname: this.options.build.publicPath || undefined,
-    exclude: [],
-    routes: this.options.generate.routes || [],
-    cacheTime: 1000 * 60 * 15,
-    filter: undefined,
-    gzip: false,
-    defaults: {}
+  let sitemaps = []
+  if (Array.isArray(this.options.sitemap)) {
+    sitemaps = this.options.sitemap
+  } else {
+    sitemaps.push(this.options.sitemap)
```

Comment:
`const sitemaps = Array.isArray(this.options.sitemap) ? this.options.sitemap : [this.options.sitemap];`